### PR TITLE
config/add font and colors to tailwind

### PR DESCRIPTION
**Pull request by CJ-Rose**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -17,6 +17,41 @@ module.exports = {
       },
     },
     extend: {
+      fontFamily: {
+        sans:["Plus Jakarta Sans",]
+      },
+      colors: {
+        // main text 'CloudBurst'
+        textBlue: "#252F56",
```
Comment (thebeyondr) [inline tailwind.config.js:25]: I think we should name our custom tokens with a prefix (eg. `vd-*`) to distinguish them. We should also maintain the tailwind kebab-case for consistency.

Examples:

- `vd-text-blue`
- `vd-beige-400`

What do you think?

File: app/root.tsx [hunk 2]
```diff
@@ -15,6 +15,10 @@ export default function App() {
 			<head>
 				<meta charSet="utf-8" />
 				<meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@200..700&display=swap"
```
Comment (thebeyondr) [inline app/root.tsx:20]: The most ideal option for us at PSE is usually to self-host the fonts. (download them and add them to our repo). 

However, we tend to use [Fontsource](https://fontsource.org/) or [Bunny Fonts](https://fonts.bunny.net/) if we're going to go the CDN route as they respect privacy more than Google! (or so they claim)

**Review by thebeyondr — CHANGES_REQUESTED** [review]
So I pulled this and the colors worked but the font didn't change. I'll leave this for you to find and fix. 

I'm confident it isn't a faulty config or conflict on my part so I think it just wasn't properly tested _before_ the review process. 

We aren't doing test-driven development but we still need to check that our implementations work for sure. happy hunting!

**Comment by CJ-Rose** [comment]
> So I pulled this and the colors worked but the font didn't change

@thebeyondr did you try removing the remix default styling on the main div of `_index.tsx`? The inline style there overrides the font set in `tailwind.css.`
I debated about including changes to that file that were only for testing purposes, and ultimately decided to leave it out.

**Comment by thebeyondr** [comment]
> @thebeyondr did you try removing the remix default styling on the main div of `_index.tsx`? 

I already knew that's what it was, but I'm encouraging you to fully complete your tasks as I'll be trusting you to make feature complete things in the future. 💪🏽

You don't have to include the tests, remove the tests and keep the feature! However, if I'm asked to review a feature and there is no feature, I'll assume there was no test.

**Review by thebeyondr — APPROVED** [review]

**Pull request merged**